components/styles.js

```diff
@@ -76,16 +75,19 @@ const styles = StyleSheet.create({
     backgroundColor: 'transparent',
     width: 28,
     height: 28,
+    left: 13,
+    top: 13,
     borderRadius: 14,
+    position: 'absolute',
```

Comment:
my bad bro. I miss read, this style is for dayCircleFiller. I would prefer to not use absolute on a non-absolute element though. You can usually achieve the same thing in a prettier way with margins and padding. Are you trying to center it?